src/box.ts

```diff
@@ -45,46 +56,49 @@ export class BoxPlot extends AVisInstance implements IVisInstance {
   }
 
   get rawSize(): [number, number] {
-    return [300, 50];
+    return [ this.options.width || 300, this.options.heightTo||50];
   }
 
   get node() {
     return <Element>this.$node.node();
   }
 
   private build($parent: d3.Selection<any>) {
-    const size = this.size,
+    const size = [this.options.width,this.options.heightTo],
       data = this.data;
     const $svg = $parent.append('svg').attr({
-      width: size[0],
-      height: size[1],
+      width: this.options.width,
+      height: this.options.heightTo,
       'class': 'phovea-box'
     });
 
-    const $t = $svg.append('g');
+    let height = this.options.width/5>=20?this.options.width/5:20;
+    height = height>this.options.heightTo?this.options.heightTo:height;
 
-    const s = this.scale = d3.scale.linear().domain((<any>this.data.desc).value.range).range([0, size[0]]).clamp(true);
+
+    const $t = $svg.append('g');
+    const s = this.scale = d3.scale.linear().domain((<any>this.data.desc).value.range).range([0.1*size[0], 0.9*size[0]]).clamp(true);
 
     $t.append('path').attr({
-      d: 'M0,0 L0,$ M0,§ L%,§ M%,0 L%,$'.replace(/%/g, String(size[0])).replace(/\$/g, String(size[1])).replace(/\§/g, String(size[1] / 2)),
+      d: 'M&,@ L&,$ M&,§ L%,§ M%,@ L%,$'.replace(/\&/g,String( 0.1*size[0])).replace(/\%/g, String(0.9*size[0])).replace(/\$/g, String(size[1]/2+ height/2)).replace(/\§/g, String(size[1] / 2)).replace(/\@/g, String(size[1]/2 - height/2)),
```

Comment:
> magic constants: `0.1` and `0.9`
I don't want the box-plot to take whole width of the column  - the whiskers are then hardly visible because they optically merge with the edge of neighboring columns. The magic constants ensure the offset. Should I declare them somewhere?